src/views/v2/commerce/gateway/stripe/payment_element.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Tickets Commerce: Checkout Buttons for Stripe
```

Comment:
We need to update the description. The file name as well, we use dashes and not underscores for separators.